docs/community/com-query/grafana-connector.md

```diff
@@ -51,26 +58,26 @@ From `grafana-7-x.x/` run the following command:
 ### Add Datasource
 
 - Sign in or create credentials (you can use admin/admin if logging in for the first time)
```

Comment:
Creating credentials is ultimately outside the scope so would belong under a "Before you begin" heading (complete with a hyperlink to help the user do this)

Reply:
@rachithrr you can skip this